notebooks/official/custom/custom_training_container_and_model_registry.ipynb

```diff
@@ -749,107 +720,316 @@
+          ]
+        }
+      ],
+      "source": [
+        "import sys\n",
+        "\n",
+        "IS_COLAB = \"google.colab\" in sys.modules\n",
+        "\n",
+        "if not IS_COLAB:\n",
+        "    ! docker build ./ -t $IMAGE_URI\n",
+        "    ! docker push $IMAGE_URI\n",
         "\n",
+        "else:\n",
+        "    ! gcloud builds submit --region={LOCATION} --tag={IMAGE_URI}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tarball_training_script"
+      },
+      "source": [
+        "#### Store training script on your Cloud Storage bucket"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ee82db61684e"
+      },
+      "source": [
+        "Exit the directory containing the Dockerfile"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 47,
+      "metadata": {
+        "id": "287dcf510706"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "/home/jupyter\n"
+          ]
+        }
+      ],
+      "source": [
+        "cd .."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ffff1d8b23fd"
+      },
+      "source": [
         "Next, you package the training folder into a compressed tar ball, and then store it in your Cloud Storage bucket."
```

Comment:
remove "you" and change "it" to "the package:

- Next, package the training folder into a compressed tar ball, and then store the package in your Cloud Storage bucket.

Reply:
Did required changes